src/Discovery/DiscoverySummary.tsx

```diff
@@ -21,7 +21,16 @@ const renderAggregation = (aggregation: AggregationConfig, studies: any[] | null
   switch (type) {
   case 'sum': {
     // parse any string representation of an integer
-    fields = fields.map((item) => (typeof item === 'string' ? parseInt(item, 10) || 0 : item));
+    // if it's not a number, return 0 so as not to break the sum
+    fields = fields.map((item) => {
+      if (typeof item === 'number') {
+        return item;
+      }
+      if (typeof item === 'string') {
```

Comment:
Nitpick: changing this to an ```else if``` is more efficient and reduces number of comparisons executed (but practically this doesn't matter).

Reply:
Since I added handling of an array and it's a return. I left it as 3 if..then blocks